src/thread/mutex_win32.cr

```diff
@@ -0,0 +1,8 @@
+# TODO: Implement for multithreading.
+class Thread
+  class Mutex
```

Comment:
If it's not implemented properly, it should be in the stubs file and the real code shouldn't be required in the prelude